acq4/devices/AxoPatch200/AxoPatch200.py

```diff
@@ -126,6 +131,43 @@ def __init__(self, dm, config, name):
         else:
             self.hasSecondaryChannel = False
         
+        if 'AxoPatchVersion' in config :
+			if config['AxoPatchVersion'] == '200A':
```

Comment:
Replace tabs with 4 spaces. Most code editors can be configured to do this by default.
See: 
https://www.python.org/dev/peps/pep-0008/#id12
https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces